source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -9,7 +9,8 @@ Test Setup
 ----------
 
 Start a three-node replica set on localhost, on ports 27017, 27018, and 27019,
-with replica set name "repl0".
+with replica set name "repl0". The replica set MUST be started with SSL
```

Comment:
There's a small problem here. The test that sets `ssl=false` in the connection string, overriding the default, will fail against this configuration.

I worked around this by only running the test where the SSL configuration of the replica set matches what's required by the URI.  So it ends up that most of the tests run against the ssl/replica_set variant, while the one that overrides ssl runs against the nossl/replica_set variant.

I'm not particularly happy with it, but it works.